edge-hub/src/Microsoft.Azure.Devices.Routing.Core/endpoints/SyncEndpointExecutor.cs

```diff
@@ -45,7 +45,7 @@ public SyncEndpointExecutor(Endpoint endpoint, ICheckpointer checkpointer, Endpo
 
         public EndpointExecutorStatus Status => this.machine.Status;
 
-        public async Task Invoke(IMessage message)
+        public async Task Invoke(IMessage message, uint _, uint __)
```

Comment:
please name these arguments

Reply:
these won't be used ever

---
In reply to: [377458446](https://github.com/Azure/iotedge/pull/2469#discussion_r377458446) [](ancestors = 377458446)